chemprop/uncertainty/predictor.py

```diff
@@ -19,8 +23,27 @@ def _calc_prediction_uncertainty(
     ) -> tuple[Tensor, Tensor]:
         """
         Calculate the uncalibrated predictions and uncertainties for the dataloader.
+
+        dataloader: DataLoader
+            the dataloader used for model predictions and uncertainty predictions
+        models: Iterable[MPNN]
+            the models used for model predictions and uncertainty predictions
+        trainer: pl.Trainer
+            an instance of the :class:`~lightning.pytorch.trainer.trainer.Trainer` used to manage model inference
+
+        Returns
+        -------
+        preds : Tensor
+            the model predictions, with shape varying by task type:
+
+            * regression/binary classification: ``n x t x m``
+
+            * multiclass classification: ``n x t x c x m``
+            where ``n`` is the number of inputs, ``t`` is the number of tasks, ``c`` is the number of classes, and ``m`` is the number of models.
```

Comment:
A broader point here, but IMO the models dimension should be the 0th dimension because it's significantly more efficient to stack big arrays along an outer dimension than an inner one.
```suggestion
            * multiclass classification: ``n x t x c x m``, where ``n`` is the number of inputs, ``t`` is the number of tasks, ``c`` is the number of classes, and ``m`` is the number of models.
```

Reply:
Sounds good! I have changed it.